### PR TITLE
feat: Close WalletConnect session on logout

### PR DESCRIPTION
**Pull request by Chef-Cheems**

### CODE REVIEW

File: src/hooks/useAuth.ts [hunk 1]
```diff
@@ -61,6 +61,9 @@ const useAuth = () => {
   const logout = useCallback(() => {
     dispatch(profileClear())
     deactivate()
+    if (connectorsByName.walletconnect) {
```
Comment (memoyil) [inline src/hooks/useAuth.ts:64]: I think this returns you a constant string not the current connector

**Comment by memoyil** [comment]
I have one pending pr in uikit https://github.com/pancakeswap/pancake-toolkit/pull/78
But your solution is much better and concise.

File: src/hooks/useAuth.ts (hunk 1 above)
Comment (memoyil) [reply src/hooks/useAuth.ts:64]: One thing you can do is 

if (window.localStorage.getItem(ConnectorNames.WalletConnect)) {

Comment (Chef-Cheems) [reply src/hooks/useAuth.ts:64]: yeah it does, this is more of a check if its initialized or not.
I'm thinking of getting current connector but haven't looked yet how to do it without modifying UIKit. For now mostly made PR for preview link :)

**Comment by Chef-Cheems** [comment]
@memoyil oh so you did make a PR for that, I thought I've seen something but couldn't find it 😄  Will check it out. I see that you manually remove the localStorage item for wallet connect, `close()` does this but also I think signals TrustWallet that connection was closed

**Comment by memoyil** [comment]
> @memoyil oh so you did make a PR for that, I thought I've seen something but couldn't find it 😄 Will check it out. I see that you manually remove the localStorage item for wallet connect, `close()` does this but also I think signals TrustWallet that connection was closed

Yes, and it also closes the websocket connection, I was doing with restarting the webpage :). Very good!

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**